canonical-web-and-design/maas-ui
# Add component docs

Done:
- Add styleguidist and basic generic component docs.

QA:
- Run `./run exec --expose-port 6060 yarn docs`.
- You should see docs for components.
